x-pack/plugins/infra/server/lib/domains/log_entries_domain/log_entries_domain.ts

```diff
@@ -150,10 +150,14 @@ export interface LogEntriesAdapter {
     start: number,
     end: number,
     bucketSize: number,
-    filterQuery?: string
+    filterQuery?: LogEntryQuery
   ): Promise<InfraDateRangeAggregationBucket[]>;
 }
 
+export interface LogEntryQuery {
+  [key: string]: any;
```

Comment:
Why `any`?

Reply:
Because I was too lazy to create a type that can accurately represent a full Elasticsearch query, which effectively is all of the Elasticsearch query DSL with all possible aggregations. That would be a project on its own.